src/page/ShopRegistration/view/Mobile/Main/index.tsx

```diff
@@ -46,11 +53,28 @@ export default function Main() {
           accept="image/*"
           id="mainMenuImage"
           className={styles['form__upload-file']}
-          onChange={saveImgFile}
+          {...register('image_urls', { required: true })}
           ref={imgRef}
+          onChange={saveImgFile}
         />
-        {imageUrls
-          ? <img src={imageUrls[0]} className={styles['form__main-menu']} alt="메인 메뉴" />
+        {imageUrls.length !== 0
+          ? (
+            <div className={styles['form__main-menu']}>
+              {imageUrls.map((image:string) => (
+                <>
+                  <img key={image} src={image} className={styles['form__main-menu-image']} alt="대표 이미지" />
+                  <button
+                    type="button"
+                    onClick={() => handleDeleteImage(image)}
+                    className={styles['form__delete-img-button']}
+                    aria-label="Delete image"
```

Comment:
```suggestion
              {imageUrls.map((url: string) => (
                <>
                  <img key={url} src={url} className={styles['form__main-menu-image']} alt="대표 이미지" />
                  <button
                    type="button"
                    onClick={() => handleDeleteImage(url)}
                    className={styles['form__delete-img-button']}
                    aria-label="Delete image"
```
띄어쓰기를 하고 url로 변수명을 바꿨어요
그리고 클래스명이 어떤 부분은 img 어떤 부분은 image라고 되어 있는데 조금 어색해서 바꾸면 더 좋을 것 같아요!